app/src/atoms/SoftwareKeyboard/AlphanumericKeyboard/index.tsx

```diff
@@ -2,18 +2,23 @@ import * as React from 'react'
 import Keyboard from 'react-simple-keyboard'
 import { alphanumericKeyboardLayout, customDisplay } from '../constants'
 
+import '../index.css'
+import './index.css'
+
+// Note (kk:04/05/2024) add debug to make debugging easy
```

Comment:
```suggestion
// TODO (kk:04/05/2024) add debug to make debugging easy
```
Mark this as a TODO?